.github/workflows/release-assignment.yml

```
@@ -74,17 +77,56 @@ jobs:
+            };
+
+            const events = await ical.async.fromURL(
+              `https://calendar.google.com/calendar/ical/${{ secrets.RELEASE_CALENDAR_ID }}/public/basic.ics`
+            );
+
+            const calendarEvents = Object.values(events)
+              .filter(event => event.type === 'VEVENT')
+              .filter(event => event.summary && event.start);
+
+            for (const [eventKey, eventTitle] of Object.entries(releaseEvents)) {
+              const matchingEvent = calendarEvents.find(event => 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)